### DISCUSSION TITLE
Trying to get my head around Archive and Categories menu items

### DISCUSSION THREAD

**Discussion by janban-outlook** [squidfunk/mkdocs-material#6741] [General]
[example.zip](https://github.com/squidfunk/mkdocs-material/files/14158450/example.zip)
[example2.zip](https://github.com/squidfunk/mkdocs-material/files/14158451/example2.zip)

I am trying to understand how the Archive and Categories menu items work.
On the blog site of mkdoc-materials these menu items are displayed in the side menu:
![image](https://github.com/squidfunk/mkdocs-material/assets/43573150/12b6ad22-910d-4d04-8a8c-ed6f058ecef1)

On my own website however, they are rendered in the top menu:
![image](https://github.com/squidfunk/mkdocs-material/assets/43573150/2aae2057-1089-4f9a-b0c1-c0e1b5769d35)

I tried to make a simple example, but that didn't make things more clear.
See the attached example sites.

In example.zip, the blog is set up as the home page. The categories and archive menus are rendered in the top bar, just as it happened on my own website.
In example2.zip, I added the blog next to a documentation site, like the mkdocs-material web site. But then the Archive and Categories menus disappear alltogether...

Please explain me what I am missing. Maybe I misunderstood the documentation.

**Comment by alexvoss** [comment]
It is best to debug this by building the subprojects individually and checking they each work. In the second example, for the German version you just need to turn the navigation entry for the blog into a list element. So, instead of `- Blog: blog/index.md` it would need to be:

```yaml
  - Blog: 
    - blog/index.md
```

The English version does not have a navigation entry for the blog, which is why no navigation entries are produced. The [documentation for the blog plugin and navigation](https://squidfunk.github.io/mkdocs-material/plugins/blog/#navigation) does say that if you have a `nav` entry you need to put an entry for the blog into it.

**Reply by janban-outlook** [reply]
I never realized the impact of making that extra level in the navigation. Now I've seen the difference it is evident, I should have realized this earlier :stuck_out_tongue:  

When having the Blog as home page, then I should do this:

```
- Blog:
  - index.md
```

**Reply by janban-outlook** [reply]
Cool. It's fixed. So simple...
![image](https://github.com/squidfunk/mkdocs-material/assets/43573150/c5b7f34e-e830-49da-b4c3-bc233841a52b)
